OpenRA.Game/Network/OrderManager.cs

```diff
@@ -182,7 +182,7 @@ public void ReceiveImmediateOrders(int clientId, OrderPacket orders)
 		public void ReceiveOrders(int clientId, (int Frame, OrderPacket Orders) orders)
 		{
 			if (pendingOrders.TryGetValue(clientId, out var queue))
-				queue.Enqueue((orders.Frame, orders.Orders));
+				queue.Add(orders.Frame, orders.Orders);
```

Comment:
do we want to check wether we are adding legacy frames? As in frames that have already passed `NetFrameNumber` as that check was removed from below

Reply:
good question.